vmdb/app/models/netapp_rcu.rb

```diff
@@ -26,6 +27,7 @@ def self.find_rcu_vcs
 
   def self.add_from_ems(ems)
     raise "NetappRcu.add_from_ems: unsupported ems type: #{ems.type}" unless ems.kind_of?(EmsVmware)
+    # TODO: Use hostname, not ipaddress
```

Comment:
Yes, I started to change them all but there were some complexities with changing them such as changing what looks like a constructor: `NetappRemoteService.add`, to make hostname required and ipaddress not required.  I tabled it and chatted with @roliveri who said that some of this code is not even used right now.  

My preference is to delete it all so no one assumes this code works and waste time changing it and instead we can resurrect it with git when we are to come back to this and make it work.  We have over 50 contributors to this project who may stumble on this code in `git grep` or elsewhere and have no idea that the code is either partially or completely unused.  Additionally, I have no idea when this code will be resurrected so we'll be carrying partially dead code until this is fixed.

Can you tell that commenting all over this code was my last choice?